src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from 'big';
+
+const operate = (numberOne, numberTwo, operation) => {
+  switch (operation) {
+    case '-':
+      return Number(Big(numberOne)) - Number(Big(numberTwo));
+    case '+':
+      return Number(Big(numberOne)) + Number(Big(numberTwo));
+    case '/':
+      return Number(Big(numberOne)) / Number(Big(numberTwo));
+    case '/0':
+      return '0';
```

Comment:
we shall check this on next milestone, ideally, you'll look for a way to tell user that this an error, You might print on the display something like "division by 0 error"

or some other creative idea